project-team-log/ProjectSuggestions.md

```diff
@@ -1,25 +1,25 @@
 # Middleware API for managing customers' loyalty points
 ### 1. Motivation
-We might assume the team is developing a middleware microservice to maintatin customers' loyalty program points.
-The Customers' Loyalty Points API (CLP API) communicates with other microservices, encapsulating the loyalty program logic.
-CL API will maintain various access levels for different application, which will enhance application's security.
+We are developing a middleware microservice, the Customers' Loyalty Points API (CLP API), to enhance customer retention and loyalty in the business strategy. This API serves as a centralized system to manage loyalty points.
+It communicates with other microservices, encapsulating the loyalty program logic.
+CLP API maintains various access levels for different applications, which enhances application's security.
 ### 2. Supported functionality
-1. Admin application is initialised in the DB (has access level `MasterApp`).
-2. Admin application can create an application and specify access level (`Can_Read` | `Can_Modify_Orders`).
-3. When application is created, the CLP API issues AppID and a token to authenticate the application later.
-4. Applications with `Can_Read` access level can (Priority - 2):
-   1. Get the customer's balance by sending a POST request with the only field:
+1. Admin application is initialised in the DB. It has access level `MasterApp`.
+2. Admin application can create applications (which represent other microservices) with specific access levels: `Can_Read`, `Can_Modify_Orders`.
+3. On creation, applications receive an AppID and a token for future authentication.
+4. Applications with `Can_Read` access level can:
+   1. Request the customer's balance by sending a GET request with the URL parameter:
       1. UserID - int
-5. Applications with `Can_Modify_Orders` access level can (Priority - 1):
+5. Applications with `Can_Modify_Orders` access level can:
    1. Insert an order into order history by sending a POST request and modify current loyalty program balance with the following structure:
       1. UserID - int
       2. OrderID - int
       3. TotalCost - int
       4. CompletedAt - int (timestamp)
-   2. Delete an order by sending a DELETE request to with the following structure:
-      5. UserID - int
-      6. OrderID - int
-6. Application with `MasterApp` access level can (Priority - 0):
+   2. Delete an order by sending a DELETE request with the following structure:
+      1. UserID - int
```

Comment:
Please update this route to match the endpoint specification

Reply:
Fair enough!